packages/gatsby-plugin-image/src/image-utils.ts

```diff
@@ -4,7 +4,21 @@ import { IGatsbyImageData } from "."
 import type sharp from "gatsby-plugin-sharp/safe-sharp"
 
 const DEFAULT_PIXEL_DENSITIES = [0.25, 0.5, 1, 2]
-const DEFAULT_BREAKPOINTS = [750, 1080, 1366, 1920]
+export const DEFAULT_BREAKPOINTS = [750, 1080, 1366, 1920]
+export const EVERY_BREAKPOINT = [
```

Comment:
Is our default for hooks really going to be all of these?

Reply:
It's for plugins that don't do any actual resizing, so passing more breakpoints just makes it more likely that there will be one that is the size of the screen

Reply:
Ah, that makes sense.